src/annotation.js

```diff
@@ -66,8 +67,15 @@ export class Annotation extends Modifier {
 
     let width = 0;
     for (let i = 0; i < annotations.length; ++i) {
```

Comment:
This could get very costly for large scores -- I think you could stick a cache in `TextFont` indexed by `hash(family+size+weight+text)`, which you can check before calculating the width.

Reply:
great idea

Reply:
2 questions: 1) where is the distance between staves (10) defined?  I want to add a line to the annotation if the font height exceeds it.   Right now I added a constant in Annotation but I figure there's a better place. 2) Web fonts don't seem to work for canvas.  Is that expected?

Reply:
OK, this is done.  Each glyph lookup is O(1) so it may not be such an improvement.